PROPOSED-fsmon-config.plist

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Version</key>
+	<string>v0.2-PROPOSED</string>
+	<key>WatchItems</key>
+	<dict>
+		<key>my_watcher</key>
+		<dict>
+			<key>Paths</key>
+      <array>
+        <!-- Paths can be strings with defaults set (IsPrefix=false) -->
+        <string>/Users/*/path1</string>
+        <!-- Paths can also be dicts with keys to override defautls -->
+        <!-- Currently one attribute, IsPrefix, maybe more in the future -->
+        <dict>
+          <key>Path</key>
+          <string>/Users/*/path2</string>
+          <key>IsPrefix</key>
```

Comment:
Discussed offline, but putting here for posterity:

There is a subtle feature of the configuration around prefix paths, and paths ending with `*`. Paths ending with the glob pattern `*` will match all items at a glob path at a point in time, and all those matches will be watched. Separately, `IsPrefix` being set would have all of the paths matching the glob be prefix paths, and anything under those paths would also match.

For example, the config:
```
<dict>
    <key>Path</key>
    <string>/Users/FakeUser/D*</string>
    <key>IsPrefix</key>
    <true/>
</dict>
```
When the config is applied (and at every update interval), all paths matching that glob would be watched (e.g., `/Users/FakeUser/Downloads`, and `/Users/FakeUser/Desktop`), and then any access down in a nested directory would then be monitored (e.g. an access of `/Users/FakeUser/Downloads/Unzipped/New/script.sh` would match).

Using the syntactic sugar to get this behavior, the configured path would need to be `/Users/FakeUser/D***`. I'm hesitant to support this because of how awkward the triple `*` looks, as well as potentially confusing users that globstar is supported within a path when it currently isn't (and no current plans to add support).